setup.py

```diff
@@ -138,6 +168,21 @@ def escape_macros(macros):
 # open_jtalk sources
 src_top = join("lib", "open_jtalk", "src")
 
+
+# extract dic
+filename = "dic.tar.gz"
+print(f"Downloading: {_DICT_URL}")
+urlretrieve(_DICT_URL, filename)
+print("Download complete")
+
+print("Extracting tar file {}".format(filename))
+with tarfile.open(filename, mode="r|gz") as f:
+    f.extractall(path="./")
+os.remove(filename)
+print("Extract complete")
+shutil.copytree(f"./{_dict_folder_name}", f"./pyopenjtalk/{_dict_folder_name}")
+sys.stdout.flush()
+
```

Comment:
Code dup with `pyopenjtalk/__init__.py`

Reply:
This is to avoid taking a long time when importing, and also for special environments such as cloud functions.

Reply:
Consider removing the lazy downloader in `pyopenjtalk/__init__.py`

Reply:
Sounds good to me. I just wanted to avoid duplication.